pkg/controller/nameservice/nameservice_controller.go

```diff
@@ -188,9 +188,9 @@ func (r *ReconcileNameService) updateNameServiceStatus(instance *rocketmqv1alpha
 		share.NameServersStr = nameServerListStr[:len(nameServerListStr)-1]
 		reqLogger.Info("share.NameServersStr:" + share.NameServersStr)
 
-		if len(oldNameServerListStr) < 2 {
+		if len(oldNameServerListStr) < 8 {
```

Comment:
magic number

Reply:
We will replace this magic number with a meaningful constant.